platform/view/core/manager/manager.go

```diff
@@ -242,21 +242,18 @@ func (cm *manager) InitiateViewWithIdentity(view view.View, id view.Identity, c
 }
 
 func (cm *manager) InitiateContext(view view.View) (view.Context, error) {
-	return cm.InitiateContextWithIdentity(view, cm.me())
+	return cm.InitiateContextFrom(cm.ctx, view, cm.me(), "")
 }
 
 func (cm *manager) InitiateContextWithIdentity(view view.View, id view.Identity) (view.Context, error) {
-	return cm.InitiateContextWithIdentityAndID(view, id, "")
+	return cm.InitiateContextFrom(cm.ctx, view, id, "")
 }
 
 func (cm *manager) InitiateContextWithIdentityAndID(view view.View, id view.Identity, contextID string) (view.Context, error) {
-	// Create the context
```

Comment:
I am iffy about this. Let's first make sure that it works on the CI and on the experiment before merging.

Reply:
For what it's worth, I ran this in a test application and it works. CI also passes. Do you want to test it anywhere else?